src/PhpWord/TemplateProcessor.php

```diff
@@ -274,6 +274,11 @@ public function setComplexValue($search, \PhpOffice\PhpWord\Element\AbstractElem
         $elementWriter->write();
 
         $where = $this->findContainingXmlBlockForMacro($search, 'w:r');
+
+        if($where === false) {
```

Comment:
please fix missing space after if (see travis check)

Reply:
It's done :)